templates/home/home-en.html

```diff
@@ -167,19 +167,56 @@ <h2>Your government</h2>
 		</div>
 	</div>
 </section>
+
 <section class="container gc-features">
-	<h2>Government initiatives</h2>
+	<h2><a id="gi"></a>Government initiatives</h2>
 	<div class="row wb-eqht">
-		<div class="col-sm-6">
-			<div class="well well-sm brdr-rds-0 eqht-trgt">
-				<img class="img-responsive full-width" src="../../components/gc-features/img/initiative-520x200.png" alt=""/>
+		<div class="col-sm-6" data-wb-randomize='{ "shuffle": true, "selector": "div", "toggle": "hide" }'>
+			<div class="well well-sm brdr-rds-0 eqht-trgt position-relative">
+				<img class="img-responsive full-width" src="../assets/home-feature-1-520x200.jpg" alt="">
+				<h3 class="h5"><a class="stretched-link" href="#">[Feature hyperlink text]</a></h3>
```

Comment:
Ensure that each feature do have a distinctive technical title + description in order to not only rely on the visual to understand that the randomize feature do work as expected. That fail SC 1.3.3